client/cody/src/chat/ChatViewProvider.ts

```diff
@@ -160,11 +160,20 @@ export class ChatViewProvider implements vscode.WebviewViewProvider {
                     try {
                         const doc = await vscode.workspace.openTextDocument(uri)
                         await vscode.window.showTextDocument(doc)
-                    } catch (error) {
-                        console.error(`Could not open file: ${error}`)
+                    } catch  {
+                        // Try to open the file in the sourcegraph view
+                        const sourcegraphInstanceUrl = 'https://sourcegraph.com'
```

Comment:
```suggestion
                        const sourcegraphInstanceUrl = this.serverEndpoint
```
Should open in connected instance

Reply:
@abeatrix I see, that makes sense. I thought it should always open up in the sourcegraph instance. Thanks for the review :)